aiconfig-docs/docs/getting-started.md

```diff
@@ -159,223 +200,192 @@ asyncio.run(main())
+placeholder typescript code
 ```
 
 </TabItem>
 </Tabs>
 
-:::note
-The AIConfig SDK supports [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations for prompts, models, parameters, and arbitrary metadata in the `aiconfig`.
-
-<!-- For more details, see the [SDK Overview](/docs/category/sdk). -->
+**Full Script**
 
-:::
+<Tabs groupId="aiconfig-language" queryString defaultValue={constants.defaultAIConfigLanguage} values={constants.aiConfigLanguages}>
+<TabItem value="python">
 
-### 6. Open the AIConfig in AI Workbook editor.
+```python title="app.py"
+from aiconfig import AIConfigRuntime, InferenceOptions
+import asyncio
 
-We can iterate on an `aiconfig` using a notebook-like editor called an **AI Workbook**. Now that we have an `aiconfig` file artifact that encapsulates the generative AI part of our application, the application code doesn't need to change even as the `aiconfig` is updated.
+config = AIConfigRuntime.load("travel.aiconfig.json")
+inference_options = InferenceOptions(stream=True)
 
-1. Go to https://lastmileai.dev.
-2. Go to Workbooks page: https://lastmileai.dev/workbooks
-3. Click dropdown from '+ New Workbook' and select 'Create from AIConfig'
-4. Upload `travel.aiconfig.json`
+async def travelWithGPT():
+  gen_itinerary_response = await config.run("gen_itinerary", params = {"order_by" : "location"}, options=inference_options, run_with_dependencies=True)
 
-Try out the workbook playground here: **[NYC Travel Workbook](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9)**
+asyncio.run(travelWithGPT())
+config.save('updated_travel.aiconfig.json', include_outputs=True)
+```
 
-:::note
-We are working on a local editor that you can run yourself. For now, please use the hosted version on https://lastmileai.dev.
-:::
+</TabItem>
 
-<video controls><source src="https://s3.amazonaws.com/publicdata.lastmileai.com/workbook_editor_480.mov"/></video>
+<TabItem value="node">
 
+```typescript title="app.ts"
+placeholder typescript code
 ```
 
-```
+</TabItem>
+</Tabs>
 
-### Code for Getting Started
+## Edit your AIConfig
 
-Tutorial code can be found [here](https://github.com/lastmile-ai/aiconfig/tree/ad38040ec3d9f0273e006464e01e02b06f2809e9/cookbook/Getting-Started).
+You can iterate and edit your aiconfig using the AIConfig Editor. Now that we have an aiconfig file artifact that encapsulates the generative AI parts of our application, the application code doesn't need to change even as the aiconfig is updated.
 
-## Additional Guides
+1. Open your Terminal
+2. Run this command: `aiconfig edit --aiconfig-path updated_travel.aiconfig.json`
 
-There is a lot you can do with `aiconfig`. We have several other tutorials to help get you started:
+A new tab with the AIConfig Editor opens in your default browser with the prompts, chaining logic, and settings from `updated_travel.aiconfig.json`. Your edits will auto-save every 15 seconds. You can also manually save with the Save button.
```

Comment:
Should probably say at `http://localhost:8080/`

Reply:
Sg!